src/main/java/htsjdk/samtools/util/IOUtil.java

```diff
@@ -87,7 +87,10 @@
     public static final long FIVE_GBS = 5 * ONE_GB;
 
     /** Possible extensions for VCF files and related formats. */
-    public static final String[] VCF_EXTENSIONS = new String[] {".vcf", ".vcf.gz", ".bcf"};
+    public static final String VCF_EXTENSION = ".vcf";
+    public static final String BCF_EXTENSION = ".bcf";
+    public static final String COMPRESSED_VCF_EXTENSION = ".vcf.gz";
+    public static final String[] VCF_EXTENSIONS = new String[] {VCF_EXTENSION, COMPRESSED_VCF_EXTENSION, BCF_EXTENSION};
```

Comment:
You don't need `new String[]` when you're using an array literal as an initializer. You can write it like this:
```
 public static final String[] VCF_EXTENSIONS = {VCF_EXTENSION, COMPRESSED_VCF_EXTENSION, BCF_EXTENSION};
```